Gems/ScriptCanvas/Code/Editor/View/Widgets/NodePalette/GeneralNodePaletteTreeItemTypes.cpp

```diff
@@ -83,7 +83,16 @@ namespace ScriptCanvasEditor
         , m_propertyStatus(propertyStatus)
     {
         GraphCanvas::TranslationKey key;
-        key << "BehaviorClass" << className << "methods" << methodName << "details";
+
+
+        AZStd::string updatedMethodName = methodName;
+        if (propertyStatus != ScriptCanvas::PropertyStatus::None)
+        {
+            updatedMethodName = (propertyStatus == ScriptCanvas::PropertyStatus::Getter) ? "Get" : "Set";
+            updatedMethodName.append(methodName);
+        }
```

Comment:
You can save a string copy operation by just appending the methodName after this if block

```suggestion
        AZStd::string updatedMethodName;
        if (propertyStatus != ScriptCanvas::PropertyStatus::None)
        {
            updatedMethodName = (propertyStatus == ScriptCanvas::PropertyStatus::Getter) ? "Get" : "Set";
        }

        updatedMethodName += methodName;
```